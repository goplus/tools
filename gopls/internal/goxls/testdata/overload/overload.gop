Demo         //@godef("Demo",Demo__0)
Demo 100     //@godef("Demo",Demo__1)
Demo "hello" //@godef("Demo",Demo__3)

var n N
n.add 100      //@godef("add",Add__0)
n.add 100, 200 //@godef("add",Add__1)
