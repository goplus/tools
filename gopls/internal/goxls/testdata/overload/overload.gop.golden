-- Demo__0-hoverdef --
```go
func Demo__0()
```

[`main.Demo__0` on pkg.go.dev](https://pkg.go.dev/golang.org/lsptests/overload#Demo__0)
-- Demo__1-hoverdef --
```go
func Demo__1(n int) int
```

[`main.Demo__1` on pkg.go.dev](https://pkg.go.dev/golang.org/lsptests/overload#Demo__1)
-- Demo__3-hoverdef --
```go
func Demo__3(s ...string)
```

[`main.Demo__3` on pkg.go.dev](https://pkg.go.dev/golang.org/lsptests/overload#Demo__3)
-- Add__0-hoverdef --
```go
func (*N).Add__0(a int)
```
-- Add__1-hoverdef --
```go
func (*N).Add__1(a ...int)
```
